backend/src/main/java/com/ody/mate/repository/MateRepository.java

```diff
@@ -13,15 +13,17 @@ public interface MateRepository extends JpaRepository<Mate, Long> {
             from Mate mate
             join fetch mate.meeting
             join fetch mate.member
-            where mate.id = :id
+            where mate.id = :id 
+            and mate.meeting.overdue = false
             """)
     Optional<Mate> findFetchedMateById(Long id);
 
     @Query("""
             select mate
             from Mate mate
             join fetch mate.member
-            where mate.meeting.id = :meetingId
+            where mate.meeting.id = :meetingId 
+            and mate.meeting.overdue = false
             """)
     List<Mate> findAllByMeetingId(Long meetingId);
```

Comment:
[제안]

유효한 meetingId로부터 mate들을 가져온다는 걸 메서드 네이밍에 담는 건 어떨까요?

만약 어떤 repository 메서드는 overdue=false가 조건으로 걸려있고 또 다른 메서드에는 걸려 있지 않다면 직관적으로 로직을 파악하기 힘들 것 같다는 생각이 들었어요! 팀원 간에 컨벤션으로기본 쿼리 메서드 명에는 overdue =false를 일관적으로 거는 것을 디폴트로 생각하자! 라고 합의한다면 현재 메서드 명도 괜찮을 것 같아요 😄

```suggestion
    List<Mate> findAllByNotOverdueMeetingId(Long meetingId);
```

Reply:
동의합니다 !
아무래도 컨벤션으로 정하더라도 논리삭제를 적용한 엔티티 안한 엔티티들로 인해 헷갈릴 여지가 많을 수 있을 것 같아
메서드 명으로 명시하는게 좋을 것 같아요 ! 👍 

다른 메서드의 경우에는 쿼리 네이밍 메서드를 사용해 OverdueFalse라고 되어 있어 이를 맞추고자 
`findAllByOverdueFalseMeetingId`로 네이밍 적용했습니다 !